src/spryker/application/skeleton/config/Shared/config_local.php

```diff
@@ -201,8 +199,16 @@
 $config[KernelConstants::DOMAIN_WHITELIST] = [];
 
 // ---------- Namespaces
-$config[KernelConstants::PROJECT_NAMESPACE] = 'Inviqa';
 $config[KernelConstants::PROJECT_NAMESPACES] = [
     'Inviqa',
     'Pyz',
 ];
+
+// ---------- OAUTH
+$config[OauthConstants::OAUTH_CLIENT_IDENTIFIER] = 'frontend';
+// todo: convert below credentials to environment variables with encrypted secrets
+$config[OauthConstants::PRIVATE_KEY_PATH] = 'file://' . APPLICATION_ROOT_DIR . '/config/Zed/dev_only_private.key';
+$config[OauthConstants::PUBLIC_KEY_PATH] = 'file://' . APPLICATION_ROOT_DIR . '/config/Zed/dev_only_public.key';
+$config[OauthConstants::ENCRYPTION_KEY] = '6Go9YRlbogOQyHpGDkydw8ejC0Z36FboKkFL7mybT0mxB3Oz';
+$config[OauthConstants::OAUTH_CLIENT_SECRET] = 'xN7uUeDzDpYtlp6oTrtC6aHGdA9Cy0gjF6hDaatiRkNyHEoc';
+$config[AuthConstants::AUTH_DEFAULT_CREDENTIALS]['yves_system']['token'] = 'VEOJWp1gsANquUEiGT7pUJdmmS0cnLLCZZKZoiRUAGRyZtUUhNVuTUSeJ8Zz1i5N7ddwv9shJXGTwVmB';
```

Comment:
Created #446 to convert these values to secrets.

Reply:
ok, we can't release this PR as a harness version without that though

Reply:
I'll also be doing stuff for the JWT to be different for pipeline envs

Reply:
true. I'm happy to work on that ticket and create a child PR of this one, but need help to understand how to do it. 

I think I can convert these values to workspace encrypted attributes for development env, but not sure how we would generate them during project setup using some twig function, also no idea how to generate sealed secrets for them at run time.

It would be nice if we can pair on it @andytson-inviqa or @kierenevans 😄